client/my-sites/plugins/main.jsx

```diff
@@ -347,7 +347,18 @@ export default React.createClass( {
 	},
 
 	render() {
+		const selectedSite = this.props.sites.getSelectedSite();
+
 		if ( this.state.accessError ) {
+			if ( this.state.accessError.abtest &&
```

Comment:
I wonder if just the second test is not enough - if `this.state.accessError.abtest` is undefined it should still behave as expected and fail. I'm no JS expert, so I might be missing something :)